srcpkgs/fldigi/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'fldigi'
+pkgname=fldigi
+version=4.1.03
+revision=1
+archs="i686 x86_64"
```

Comment:
Why this restriction? If there is no good reason, remove this line.

Reply:
Removed the restrictions on the last set of changes.